wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
The determination if the file is local or not should not happen within the `check_template`(as you did <a href="https://github.com/blackHatMonkey/wifiphisher/commit/b3b6edaa6d471fe8370107cce732f8ea752c9267#diff-daf2e6c81475b535f8cfa782559e9d5bR207">here</a>).

It should happen in `wifiphisher.py`. If `TEMPLATE_NAME` lies in `phishingpage.TEMPLATE_DATABASE` dictionary and its value is `None` then we have the template stored locally and there's no need to run `phishingpage.check_template` in this case.